conda/environments/cudf_dev_cuda10.2.yml

```diff
@@ -14,8 +14,9 @@ dependencies:
   - cmake_setuptools>=0.1.3
   - python>=3.6,<3.8
   - numba>=0.49.0
+  - pandas >=1.0,<1.1
```

Comment:
```suggestion
```

Reply:
thanks, this probably explains why my environment was not getting set up properly.